thewalrus/symplectic.py

```diff
@@ -99,8 +100,33 @@ def expand_vector(alpha, mode, N, hbar=2.0):
     return r
 
 
+def expand_passive(T, modes, N):
+    r"""Returns and expanded linear optical transformation
+    acting on specified modes, with identity acting on all other modes
+
+    Args:
+        T (array): square :math:`M \times M` matrix of linear optical transformation
+        modes (array): :math:`M` modes of the tranformation
+        N (int): number of modes
+
+    Returns:
+        array: phase-space displacement vector of size 2*N
```

Comment:
This line does not sound right...

Reply:
good spot, I think this was copy pasted and not suitably updated!